src/coprocessor/select/select.rs

```diff
@@ -152,7 +152,12 @@ impl<'a> SelectContext<'a> {
             } else {
                 range.get_start().to_vec()
             };
-            let upper_bound = if !self.core.desc_scan && !range.get_end().is_empty() {
+            let lower_bound = if !range.get_start().is_empty() {
```

Comment:
Could we set both `lower bound` and `upper bound` like we do in function `init_with_range`?

Reply:
Is is possible the start or end is empty?